packages/cli/docker/docker-compose.yaml

```diff
@@ -20,7 +20,6 @@ services:
         environment:
```

Comment:
Do we still need this file?

Reply:
The docker-run command still exists. For internal usage I think it is worthwhile keeping. It is the easiest way to spin up a clean environment locally to test integration scripts.